modules/nw-operator-cr.adoc

```diff
@@ -23,239 +23,300 @@ endif::[]
+|`object`
+|
+The parameters for this object specify the `kube-proxy` configuration.
+If you do not specify the parameter values, the Cluster Network Operator applies the displayed default parameter values.
+If you are using the OVN-Kubernetes cluster network provider, the kube-proxy configuration has no effect.
 
-<4> The parameters for this object specify the Kubernetes network proxy (kube-proxy) configuration. If you are using the OVN-Kubernetes default CNI network provider, the kube-proxy configuration has no effect.
+|====
 
-<5> The refresh period for `iptables` rules. The default value is `30s`. Valid suffixes include `s`, `m`, and `h` and are described in the link:https://golang.org/pkg/time/#ParseDuration[Go time package] documentation.
-+
-NOTE: Because of performance improvements introduced in {product-title} 4.3 and greater, adjusting the `iptablesSyncPeriod` parameter is no longer necessary.
+[discrete]
+[id="nw-operator-cr-defaultnetwork_{context}"]
+=== defaultNetwork object configuration
 
-<6> The minimum duration before refreshing `iptables` rules. This parameter ensures that the refresh does not happen too frequently. Valid suffixes include `s`, `m`, and `h` and are described in the link:https://golang.org/pkg/time/#ParseDuration[Go time package].
-endif::operator[]
+The values for the `defaultNetwork` object are defined in the following table:
+
+.`defaultNetwork` object
+[cols=".^3,.^2,.^5",options="header"]
+|====
+|Field|Type|Description
 
+|`type`
+|`string`
+|Either `OpenShiftSDN` or `OVNKubernetes`. The cluster network provider is selected during installation. This value cannot be changed after cluster installation.
 ifdef::openshift-origin[]
 [NOTE]
 ====
 {product-title} uses the OVN-Kubernetes Container Network Interface (CNI) cluster network provider by default.
 ====
 endif::openshift-origin[]
 
+|`openshiftSDNConfig`
+|`object`
+|This object is only valid for the OpenShift SDN cluster network provider.  This value cannot be changed after cluster installation.
+
+|`ovnKubernetesConfig`
+|`object`
+|This object is only valid for the OVN-Kubernetes cluster network provider.  This value cannot be changed after cluster installation.
+
+|====
+
+[discrete]
 [id="nw-operator-configuration-parameters-for-openshift-sdn_{context}"]
-== Configuration parameters for the OpenShift SDN CNI cluster network provider
+==== Configuration for the OpenShift SDN CNI cluster network provider
 
 The following YAML object describes the configuration parameters for
-the OpenShift SDN default Container Network Interface (CNI) network provider.
+the OpenShift SDN Container Network Interface (CNI) cluster network provider.
 
+.`openshiftSDNConfig` object
+[cols=".^2,.^2,.^6a",options="header"]
+|====
+|Field|Type|Description
+
+|`mode`
+|`string`
+|
+ifndef::operator[]
+Configures the network isolation mode for OpenShift SDN. The allowed values are `Multitenant`, `Subnet`, or `NetworkPolicy`. The default value is `NetworkPolicy`.
```

Comment:
Maybe

    Configures the network isolation mode for OpenShift SDN. The default value is `NetworkPolicy`.
    The values `Multitenant` and `Subnet` are also allowed for backward-compatibility with OCP 3,
    but are not recommended.